src/main/java/io/visual_regression_tracker/sdk_java/response/VRTStopResponse.java

```diff
@@ -0,0 +1,23 @@
+package io.visual_regression_tracker.sdk_java.response;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public class VRTStopResponse {
```

Comment:
could you extend `BuildResponse` with those new fields? 
it's the same dto used for start/stop build on api side